docs/r2dbc.md

```diff
@@ -268,6 +268,24 @@ The environment's application default principal impersonates
 SERVICE_ACCOUNT_1 which impersonates SERVICE_ACCOUNT_2 which then
 impersonates the TARGET_SERVICE_ACCOUNT.
 
+## SQL Admin API URL
+
+The Java Connector supports setting the SQL Admin API URL with the
+`ADMIN_ROOT_URL` and `ADMIN_SERVICE_PATH` options. The `ADMIN_ROOT_URL`
+option specifies the URL-encoded root URL of the service and the
+`ADMIN_SERVICE_PATH` option specifies the URL-encoded service path of the
+service.
+
+### Example
+
+```java
+ConnectionFactoryOptions options=ConnectionFactoryOptions.builder()
```

Comment:
More realistic example strings for the docs: 
"https://googleapis.example.com/"
"sqladmin/v1/"

Also, we may want to direct people to see the javadoc for `setRootUrl` and `setServicePath`: https://cloud.google.com/java/docs/reference/google-api-client/latest/com.google.api.client.googleapis.services.AbstractGoogleClient.Builder#com_google_api_client_googleapis_services_AbstractGoogleClient_Builder_setRootUrl_java_lang_String_

Reply:
Updated.